force_wfmanager/ui/review/tests/test_plot.py

```diff
@@ -274,54 +287,49 @@ def test_recenter_plot(self):
         self.assertEqual(self.plot._plot.range2d.x_range.low, 1.9)
 
 
-class TestBasePlot(TestAnyPlot, unittest.TestCase):
-
-    def setUp(self):
-        super(TestBasePlot, self).setUp()
-        self.plot = BasePlot(analysis_model=self.analysis_model)
-
-
-class TestPlot(TestAnyPlot, unittest.TestCase):
-
+class TestPlot(TestAnyPlot):
```

Comment:
Does this mean that the unit tests in the `TestAnyPlot` class will be performed twice?

Reply:
I believe so, but the `setUp` method generates different `Plot` subclass.